### DISCUSSION TITLE
请问各位，tbb怎么设置生成静态库

### DISCUSSION THREAD

**Discussion by KaivnD** [xmake-io/xmake#4393] [Q&A] [answered]
如题，在用embree的时候想用静态库，找到一个config可以改

https://github.com/xmake-io/xmake-repo/blob/bb6ee959727b0776d428ebdbb0914d1a226d4d7b/packages/e/embree/xmake.lua#L15-L16C84

这里标注了一下，不建议将embree作为静态库使用，想知道为啥 😂

然后我想问，如何让embree的依赖tbb也生成静态库

**Comment by waruqi** [comment, marked as answer]
这你得问打包的人，通常禁用 要么就是当初打 静态库失败，有可能包本身的构建系统不支持，或者代码不支持等各种原因，也许打 patch 可以过掉，但是短时间内也许打包者搞不定 就暂时禁用了。。

要么就是这个包 官方只推荐使用动态库。。反正具体原因 你要么问打包的人，要么自己提 pr 过来 让它支持上 静态库，并且重新趟坑，只要 ci 上所有 pass 都能过，并且本地运行正常，就可以 merge
